sklearn/manifold/tests/test_mds.py

```diff
@@ -85,3 +111,37 @@ def test_normalized_stress_auto(metric, monkeypatch):
 
     mds.smacof(dist, metric=metric, normalized_stress="auto", random_state=rng)
     assert mock.call_args[1]["normalized_stress"] != metric
+
+
+def test_isotonic_outofbounds():
+    # This particular configuration can trigger out of bounds error
+    # in the isotonic regression (was fixed in PR 30514)
```

Comment:
Rather than mention the PR which can be usually be found via git blame, we usually make it explicit that this is a non-regression test for a given issue number.